en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,75 @@
+# PBKDF2 hashing
+We can use Password-Based Key Derivation Function 2 (PBKDF2) hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
+
+This guide walks you through the steps of configuring PBKDF2 as the hashing algorithm of a JDBC userstore.
+
+!!! note
+    Currently PBKDF2 supports only JDBC userstores in Asgardeo.
+
+## Prerequisite
+You should have a secondary JDBC userstore added.  [Create a JDBC user store](../configure-secondary-user-stores) if you don’t already have one.
```

Comment:
As stated in https://github.com/wso2/product-is/issues/12124#issuecomment-910492002 we do support PBKDF2 for primary user store. Better to include this information too. However note that this can be used before the initial server startup only (otherwise the admin password validation will fail).

@divyaamunugama FYI: https://github.com/wso2/product-is/issues/12394#issue-982427664

@UdeshAthukorala you implemented a utility that can be used to update the admin password right? can you provide more details on how this can be achieved.

Reply:
@dewniMW The client[1] that I created is used to generate a PBKDF2 Hash and the Salt Value for a given plain text password. If we know the password we can use this client to generate pbkdf2 hash and the salt value and update database directly.

[1]https://github.com/wso2-extensions/identity-hash-provider-pbkdf2/tree/main/utility/pbkdf2-hash-generator